scss/util/_color.scss

```diff
@@ -4,7 +4,7 @@
 
 @import 'math';
 
-/// Colors available by using `get-color()`.
+/// Colors available by using `get-color()`. These should not be used directly within components, variables should be used instead to make components customizable.
```

Comment:
`get-state() and settings should be used instead to make components customizable.`